src/database/inquirer_database_script.sql

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Move database/inquirer_database_script.sql to root folder. src folder should contain only main and test folders according maven specification